code/mob/living/carbon/human.dm

```diff
@@ -681,6 +681,8 @@
 				HS.changeling.transferOwnership(HS)
 				HS.changeling.owner = HS
 				HS.changeling.reassign_hivemind_target_mob()
+				var/obj/head = src.organHolder.drop_organ("head")
```

Comment:
path this

Reply:
There's really no reason to since none of its members are accessed. It's more a style thing, you gotta choose between that or just qdel(src.organholder.grop_organ("head")).

Reply:
yes

Reply:
Hopefully done correctly